pkg/kubelet/apis/config/fuzzer/fuzzer.go

```diff
@@ -62,6 +62,7 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 			obj.KernelMemcgNotification = false
 			obj.MaxOpenFiles = 1000000
 			obj.MaxPods = 110
+			obj.MemoryManagerPolicy = "none"
```

Comment:
Is it possible to use the defined type we have for this instead of the unqualified string "none"?

Reply:
Is this also a case, where we should probably have entries for the CPUManager and TopologyManager in the long run?

Reply:
done

Reply:
We already have them under:
- CPU manager -https://github.com/kubernetes/kubernetes/blob/f170d33e1f320d7a3bcd2ae5a60b7d335ae382f6/pkg/kubelet/apis/config/fuzzer/fuzzer.go#L69
- Topology Manager - https://github.com/kubernetes/kubernetes/blob/f170d33e1f320d7a3bcd2ae5a60b7d335ae382f6/pkg/kubelet/apis/config/fuzzer/fuzzer.go#L72

Reply:
Ack